packages/editor/src/components/post-slug/test/index.js

```diff
@@ -11,17 +11,15 @@ import { PostSlug } from '../';
 
 describe( 'PostSlug', () => {
 	it( 'should update slug with sanitized input', async () => {
-		const user = userEvent.setup( {
-			advanceTimers: jest.advanceTimersByTime,
-		} );
+		const user = userEvent.setup();
 		const onUpdateSlug = jest.fn();
 
 		render( <PostSlug postSlug="index" onUpdateSlug={ onUpdateSlug } /> );
 
 		const input = screen.getByRole( 'textbox', { name: 'Slug' } );
 		await user.clear( input );
 		await user.type( input, 'Foo Bar-Baz 9!' );
-		input.blur();
+		act( () => input.blur() );
```

Comment:
Maybe we could just trigger a `tab` and that'd blur it anyway, but in a way resembling user behavior more closely?